blockchain.com/src/endpoint/height.ts

```diff
@@ -0,0 +1,27 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { Config, ExecuteWithConfig } from '@chainlink/types'
+
+export const Name = 'height'
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  console.log('EXECUTING!')
```

Comment:
These are going to gum up the log files since they log with every request. Use `logger.debug` here instead.